docs/source/index.rst

```diff
@@ -1,53 +1,164 @@
-of your code.
-
-The conda package and environment manager is included in all versions of
-:ref:`Anaconda <anaconda-glossary>`\ |reg|,
-:ref:`Miniconda <miniconda-glossary>`, and
-`Anaconda Repository <https://docs.continuum.io/anaconda-repository/>`_.
-Conda is also included in `Anaconda Enterprise
-<https://www.anaconda.com/enterprise/>`_, which provides on-site enterprise
-package and environment management for Python, R, Node.js, Java, and other
-application stacks. Conda is also available on
-`conda-forge <https://anaconda.org/conda-forge/conda>`_, a community channel.
-You may also get conda on `PyPI <https://pypi.org/>`_, but
-that approach may not be as up to date.
+===================
+Conda Documentation
+===================
+
+Welcome to conda's documentation! Conda provides package, dependency, and environment
+management for any language. Here, you will find everything you need to get started
+using conda in your own projects.
+
+Install  :octicon:`download;1em;sd-text-primary`
+................................................
+
+We recommend the following methods to install conda:
+
+.. tab-set::
+
+    .. tab-item:: Windows :fab:`windows`
+
+        Miniconda installer for:
+
+        .. grid:: 2
+
+            .. grid-item::
+
+                .. button-link:: https://repo.anaconda.com/miniconda/Miniconda3-latest-Windows-x86_64.exe
+                    :color: primary
+
+                    Windows x86 64-bit :octicon:`download`
+
+    .. tab-item:: MacOS :fab:`apple`
+
+        Miniconda installer for:
+
+        .. grid:: 2
+
+            .. grid-item::
+
+                .. button-link:: https://repo.anaconda.com/miniconda/Miniconda3-latest-MacOSX-x86_64.pkg
+                    :color: primary
+
+                    MacOS x86 64-bit :octicon:`download`
+
+            .. grid-item::
+
+                .. button-link:: https://repo.anaconda.com/miniconda/Miniconda3-latest-MacOSX-arm64.pkg
+                    :color: primary
+
+                    MacOS M1 64-bit :octicon:`download`
+
+    .. tab-item:: Linux :fab:`linux`
+
+        Miniconda installer for:
+
+        .. grid:: 2
+
+            .. grid-item::
+
+                .. button-link:: https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh
+                    :color: primary
+
+                    Linux x86 64-bit :octicon:`download`
+
+    .. tab-item:: Homebrew :fa:`beer-mug-empty`
+
+        Run the following `Homebrew <https://brew.sh/>`_ command:
+
+        .. code-block:: bash
+
+            brew install miniconda
+
+    .. tab-item:: Chocolatey :octicon:`terminal` :fab:`windows`
+
+        Run the following `Chocolatey <https://chocolatey.org/>`_ command:
+
+        .. code-block:: bash
+
+            choco install miniconda3
+
+
+.. raw:: html
+
+    <p class="text-small">For more detailed instructions, see <a href="https://docs.conda.io/projects/miniconda/en/latest/">Miniconda's installation guide</a></p>
```

Comment:
Please don't link to specific language and version builds of the docs, this makes changing configuration on the RTD much harder

```suggestion
    <p class="text-small">For more detailed instructions, see <a href="https://docs.conda.io/projects/miniconda/">Miniconda's installation guide</a></p>
```